### PR TITLE
[Serializer] Groups annotation/attribute on class

### LINKED ISSUE #18653: [Serializer] Groups annotation/attribute on class
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#49594
| PR author(s) | @Brajk19
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/18653

### CODE REVIEW

File: serializer.rst [hunk 1]
```diff
@@ -321,7 +321,37 @@ to your class::
         private string $description;
     }
 
-You can now choose which groups to use when serializing::
+You can also add the ``#[Groups]`` attribute on your class directly::
```
Comment (OskarStark) [inline serializer.rst]:
```suggestion
You can also use the ``#[Groups]`` attribute on class level::
```

File: serializer.rst [hunk 2]
```diff
@@ -321,7 +321,37 @@ to your class::
...
+        private string $description;
+    }
+
+In this example, the ``id`` and the ``name`` properties belong to the
+``show_product`` and ``list_product`` groups. The ``description`` property
+only belongs to the ``show_product`` group.
+
+.. versionadded:: 6.4
+
+    The support of the ``#[Groups]`` attribute on the class itself was
```
Comment (OskarStark) [inline serializer.rst]:
```suggestion
    The support of the ``#[Groups]`` attribute on class level was
```

**Review by OskarStark — APPROVED** [review]

**Comment by alexandre-daubois** [comment]
Comments addressed :ok_hand:

**Comment by OskarStark** [comment]
Thank you Alexandre.

**Pull request merged**